README.rst

```diff
@@ -106,6 +106,20 @@ otherwise, add the include and library paths in the environment variables ``TORC
 .. _libjpeg: http://ijg.org/
 .. _libjpeg-turbo: https://libjpeg-turbo.org/
 
+Video Backend
+=============
+Torchvision currently supports the following video backends:
+
+* pyav (default) - Pythonic binding for ffmpeg libraries. 
```

Comment:
Let's  add a link so that users can read instructions on how to install the package, etc.

```suggestion
* [pyav](https://github.com/PyAV-Org/PyAV) (default) - Pythonic binding for ffmpeg libraries. 
```

Reply:
Makes sense, thanks!